default.env

```diff
@@ -75,3 +75,11 @@ export KLOG_VERBOSITY='1'
 # Example: 5d
 # Required: no
 export IMAGE_TAG_EXPIRATION='6h'
+
+# A github appid used to set up Pac integerating with Github App
+# Required: no
+export PAC_GITHUB_APP_ID=''
+
+# # A github private key used to set up Pac integerating with Github App
+# Required: no
+export PAC_GITHUB_APP_PRIVATE_KEY=''
```

Comment:
should these be `E2E_PAC_GITHUB_APP_ID` and `E2E_PAC_GITHUB_APP_PRIVATE_KEY` instead? (since those without `E2E_` prefix get overridden [here](https://github.com/redhat-appstudio/e2e-tests/pull/703/files#diff-be9cd886dab1d153a199cd4e8e53764a124d2bb335339a13d24993b774d0d0f7R126))

Reply:
fixed